sideboard/lib/_redissession.py

```diff
@@ -0,0 +1,88 @@
+import threading
+
+try:
+  import cPickle as pickle
+except ImportError:
+  import pickle
+
+from cherrypy.lib.sessions import Session
+import redis
+from redis import Sentinel
+
+class RedisSession(Session):
```

Comment:
... should I use this when I integrate redis with Uber to store global session vars?

Reply:
You can probably just use the redis library directly. In this case I ended up just inlining the cherrys package (Which previously we installed) since it's all of ~100 lines and we needed some changes.